packages/flutter/lib/src/material/calendar_date_picker.dart

```diff
@@ -315,20 +315,28 @@ class _CalendarDatePickerState extends State<CalendarDatePicker> {
     assert(debugCheckHasMaterial(context));
     assert(debugCheckHasMaterialLocalizations(context));
     assert(debugCheckHasDirectionality(context));
+    const double fontSizeToScale = 14.0;
+    final double textScaleFactor = MediaQuery.textScalerOf(context).clamp(maxScaleFactor: 3.0).scale(fontSizeToScale) / fontSizeToScale;
```

Comment:
Can you create a const for 3.0 and name it something like maxScalingFromMobileDevice and add some comment to explain how this value come from?